docs/src/main/sphinx/security/https.rst

```diff
@@ -0,0 +1,357 @@
+.. _https_tls_top:
+
+=============
+HTTPS and TLS
+=============
+
+Trino (formerly PrestoSQL) runs with no security configuration in its default
+installation. This allows you to connect to the server using URLs that specify
+the HTTP protocol when using the Trino :doc:`CLI </installation/cli>`, the
+:doc:`Web UI </admin/web-interface>`, or other clients.
+
+This topic describes how to configure your Trino server to use :ref:`TLS
+<glossTLS>` to require clients to use the HTTPS connection protocol. All
```

Comment:
As long as the :ref: links to labelled entries still work, it doesn't really matter where the Glossary is. However, I do like having this single subject matter Glossary in one place.

Reply:
Okay for now. The problem is that we end up with a bunch of glossaries that way .. but maybe that is fine. Let's leave as is. Since the ref link will continue to work we can always move later with little effort.